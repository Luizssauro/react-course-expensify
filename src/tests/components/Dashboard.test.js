import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from '../../components/Dashboard';

describe('<Dashboard />', () => {
	test('should render Dashboard component', () => {
		const wrapper = shallow(<Dashboard />);

		expect(wrapper).toMatchSnapshot();
	});
});
