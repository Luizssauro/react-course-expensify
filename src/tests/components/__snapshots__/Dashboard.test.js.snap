// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> should render Dashboard component 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
